scripts/intfutil

```diff
@@ -276,7 +276,7 @@ def po_speed_dict(po_int_dict, appl_db):
             elif len(value) > 1:
                 for intf in value:
                     temp_speed = appl_db.get(appl_db.APPL_DB, "PORT_TABLE:" + intf, "speed")
-                    temp_speed = int(temp_speed)
+                    temp_speed = int(temp_speed) if temp_speed else 0
```

Comment:
processing of the exception should protect from both empty speed and invalid speed string (e.g, 1000o)

Reply:
You are absolute right.

But I think app_db's speed is populated by SONiC, it could be missing, but shouldn't have format issue. If we did cause such a code bug, I think we should fix the formatting code.